### DISCUSSION TITLE
How to effectively delete

### DISCUSSION THREAD

**Discussion by garethsnaim** [Rem0o/FanControl.Releases#870] [General]
Can someone tell me how to delete the programme as if it had never been installed? Simply binning the folder is not it. when I reinstall it says all the sensors are missing still as if its looking at an old config?

**Comment by xhul-dev** [comment]
Hello there.
Apart from the folder itself, the only dependency it has with the system is an entry in the task scheduler, which only exists if the "start with windows" option is on.
I believe that's all there is, so you might have a separate issue.

**Comment by garethsnaim** [comment]
My problem is the pump fan. Nothing I do allows the app to control it. If I set a curve etc and set it to auto the fan runs at full speed. If I manuall control it I can drop the speed right down.

I have played in Bios as well. Should I take the pump fan off the pump header?

**Reply by Rem0o** [reply]
You could try to change header to an other one, worth a shot.
